log/lvl.go

```diff
@@ -0,0 +1,385 @@
+// Package log is an output-library that can print nicely formatted
```

Comment:
Because I re-wrote most of these to put all methods together:

- lvl.go - all Lvl.. methods
- ui.go - other log-commands
- common.go - structures and variables